drivers/client_access_control/spec/requests/client_visibility_legacy_spec.rb

```diff
@@ -29,10 +35,9 @@ def doc_has_client?(doc, warehouse_client)
 
     describe 'and the user does not have a role' do
       it 'user cannot see any clients' do
-        get clients_path(q: 'bob')
```

Comment:
Is it worth keeping one of these old get versions somewhere (maybe it is below, but I haven't gotten there yet) to confirm the redirect behavior?